src/UFX.Relay/Tunnel/Listener/TunnelConnectionListener.cs

```diff
@@ -35,7 +35,7 @@ public async Task BindAsync()
         unbindTokenSource = new CancellationTokenSource();
         endpoint = tunnelEndpoint;
         endpoint.TunnelId = await tunnelIdProvider.GetTunnelIdAsync() ?? throw new KeyNotFoundException("TunnelId not found");
-        var cts = new CancellationTokenSource(5000);
```

Comment:
Worth nothing that beyond my issue, this timeout was problematic, since it was equal to the sleep in the `WebSocketException` catch. So if it needed to retry, it would have likely timed out.